packages/main/src/Tokenizer.ts

```diff
@@ -137,10 +90,66 @@ const metadata = {
  * @usestextcontent
  * @private
  */
+@customElement("ui5-tokenizer")
+@languageAware
+
+@event("token-delete", {
+	detail: {
+		ref: { type: HTMLElement },
+	},
+})
+
+@event("show-more-items-press", {
+	detail: {
+		ref: { type: HTMLElement },
+	},
+})
+
 class Tokenizer extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	@property({ type: Boolean })
```

Comment:
Many of the properties here are used in other controls (multi input for example) and they lack documentation. Maybe we should make them public and add documentation, but I am not sure if could be done (even though the tokenizer is used only internally, it is marked as public)?

@SAP/ui5-webcomponents-core  could you please give an opinion?

Reply:
The class itself is marked as "private" - line 91,
The events are not marked as public, so they are private
The properties and slots are not marked as public as well, so they are private.
The result is - no documentation is generated for the Tokenizer.

You can as much as JSDoc as you want, no matter if it's public or private, so this is up to you.

The question is where do you want that JSDoc to be visible.
The Tokenizer is not meant for a public component, so I guess it should be remain private and not be shown in the Playground next to the other components?

Could you describe what do you expect?

Reply:
Private properties of the tokenizer are used outside of it and I am not sure if this is correct. One example is the `showMore` property, which is marked as private, but it is used in the hbs of the multiinput. If this is okay to be done this way in private components we can leave it as it is.

Reply:
we decided to handle this separately

Reply:
we decided to handle this separately